desc/objectives/linear_objectives.py

```diff
@@ -793,88 +793,36 @@ class FixLambdaGauge(_Objective):
 
     """
 
-    _scalar = False
-    _linear = True
-    _fixed = False  # not "diagonal", since it is fixing a sum
     _units = "(rad)"
     _print_value_fmt = "lambda gauge error: "
 
     def __init__(
         self,
         eq,
+        target=0,
```

Comment:
I think for the other ones we don't even allow the user to pick target bounds or weight